runtime/tools/spec-builder/index.js

```diff
@@ -5,24 +5,30 @@ const curPath = process.cwd();
 
 let argv = require('yargs')
     .usage('Usage: $0 <command> [options]')
-    .command('build-spec', 'Converts custom spec json file to raw')
-    .example('$0 build-spec -f customSpec.json -o customSpecRaw.json', 'convert given file to raw and write to output')
+    .example('$0 -f customSpec.json -o customSpecRaw.json', 'convert given file to raw and write to output')
     .alias({'f' : 'file', 'o' : 'output'})
     .nargs({'f' : 1, 'o': 1})
     .describe('f', 'Load a file')
     .demandOption(['f'])
     .help('h')
     .alias('h', 'help')
-    .epilog('copyright 2020')
+    .epilog('copyright 2020 Limehchain LTD.')
     .argv;
 
-let customSpec = require(`${curPath}/${argv.file}`);
-const rawGenesis = GenesisBuilder.toRaw(customSpec);
 
-customSpec.genesis = rawGenesis;
+if(fs.existsSync(`${curPath}/${argv.file}`)){
```

Comment:
When you are checking/making assertions with this kind of `if` it is easier for the code to be read (from a dev perspective) if you check for the "stopping"/"invalid" case at first and after that simply resume as you don't need the check. That's why I did the `ifs` in the `Balances` and `System` files at first, checked for the assertions and continues afterwards. 

Meaning that your code can be changed to:
```JavaScript
if(!fs.existsSync(`${curPath}/${argv.file}`)){
    console.log(`Error: ${argv.file} doesn't exist at the provided path: ${curPath}/${argv.file}`);
    return;
}

let customSpec = require(`${curPath}/${argv.file}`);
const rawGenesis = GenesisBuilder.toRaw(customSpec);
customSpec.genesis = rawGenesis;

// set default file to write, if output file isn't provided
const outputFile = argv.output ? argv.output : `${curPath}/customSpecRaw.json`;
fs.writeFileSync(outputFile, JSON.stringify(customSpec, null, 2));

console.log("Succesfully converted to raw json.");
```